.github/workflows/test-csharp.yml

```diff
@@ -121,17 +121,12 @@ jobs:
         uses: actions/download-artifact@v4
         with:
           name: regorus-nuget
-          path: ./bindings/csharp/TestApp/regorus-nuget/
-        
-      - name: Restore Test App
-        run: dotnet restore /p:RestoreSources=./regorus-nuget/
-        working-directory: ./bindings/csharp/TestApp
-
-      - name: Build Test App 8.0
-        run: dotnet build --framework net8.0
-        working-directory: ./bindings/csharp/TestApp
-
-      - name: Run Test App 8.0
-        run: dotnet run --framework net8.0
-        working-directory: ./bindings/csharp/TestApp
-  
\ No newline at end of file
+          path: ./bindings/csharp/Regorus.Tests/regorus-nuget/
+
+      - name: Restore Regorus.Tests
+        run: dotnet restore /p:RestoreAdditionalProjectSources=./regorus-nuget
```

Comment:
For my own understanding, why use RestoreAdditionalProjectSources instead of RestoreSources?
The documentation does not explain the rationale: https://learn.microsoft.com/en-us/nuget/reference/msbuild-targets

Reply:
It's my first time encountering these settings, but my understanding is that `RestoreSources` overwrites the list of sources, whereas `RestoreAdditionalProjectSources` adds additional sources to the list.

By default, dotnet will point to the public nuget feed (nuget.org), so if you replace the source list with a local file path, then you'll no longer be able to pull public packages.